doc/conf.py

```diff
@@ -178,7 +178,7 @@
 todo_include_todos = False
 
 intersphinx_mapping = {
-    'python': ('https://docs.python.org/2/', None),
+    'python': ('https://docs.python.org/3/', None),
```

Comment:
not sure where this is used

Reply:
Ref links to Python objects and docs, not sure if there are still extant ones.